src/librustc_mir/tcx/expr.rs

```diff
@@ -541,14 +569,16 @@ fn convert_var<'a, 'tcx: 'a>(cx: &mut Cx<'a, 'tcx>,
                     match expr.node {
                         hir::ExprClosure(_, _, ref body) => body.id,
                         _ => {
-                            cx.tcx.sess.span_bug(expr.span,
-                                              &format!("closure expr is not a closure expr"));
+                            cx.tcx
+                              .sess
+                              .span_bug(expr.span, &format!("closure expr is not a closure expr"));
```

Comment:
In these cases it won't fit with a newline after the arg, so it reverts to splitting the call chain. I don't find this too bad. Since rustfmt can't cheat like in the original, the only alternative would be to break after the `(`. In general, I think it would be bad to prefer that to breaking the chain of fields (the latter is what you most often want). So, I'm not sure how to identify this kind of special case.